bin/plugin/commands/performance.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * External dependencies
+ */
+const { v4: uuid } = require( 'uuid' );
+const path = require( 'path' );
+const os = require( 'os' );
+
+/**
+ * Internal dependencies
+ */
+const { formats, log } = require( '../lib/logger' );
+const {
+	runShellScript,
+	readJSONFile,
+	askForConfirmation,
+} = require( '../lib/utils' );
+const git = require( '../lib/git' );
+const config = require( '../config' );
+
+/**
+ * @typedef WPRawPerformanceResults
+ *
+ * @property {number[]} load Load Time.
+ * @property {number[]} domcontentloaded DOM Contentloaded time.
+ * @property {number[]} type Average type time.
+ * @property {number[]} focus Average block selection time.
```

Comment:
While we're adding more and more types, I wonder if we should abandon this rule. The added value is small IMO and it's disruptive to commits and it's also time-consuming (unless we find a way to auto-fix it).

Reply:
I think it would be a good topic to surface, possibly in a future JavaScript meeting.

There was another recent discussion of this at https://github.com/WordPress/gutenberg/pull/22077#discussion_r422275562 , where I share a few of my thoughts on the matter.

I do feel like it improves readability, though I also agree that the manual effort is excessive, but in a way which could theoretically be solved with tooling. The line length challenges are somewhat more troublesome to overcome.

On the other-hand, if we assume that most people would be "reading" these through IDE integrations of the types themselves (VSCode tooltips, etc), then maybe readability of the source isn't so much of a concern.